app/models/proxy_performance.rb

```diff
@@ -3,4 +3,10 @@ class ProxyPerformance < ActiveRecord::Base
   belongs_to :site, inverse_of: :proxy_performances
 
   include Concerns::SoftDeletable
+
+  def increment(times_succeeded:, times_failed:)
```

Comment:
Can you provide a default value with the `(option1:,option2:)` notation? If so, it might be worth defaulting both of these to zero so that you can call this method with only one of its options. (Nice to have, not strictly necessary, since the only place we call it at the moment passes both anyway).